7-alpine/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mcr.microsoft.com/dotnet/sdk:7.0-alpine
+
+# Check latest version at https://api.github.com/repos/docker/cli/tags
+ENV DOCKER_VERSION 20.10.22
```

Comment:
It appears that the latest in https://api.github.com/repos/docker/cli/tags is 23.0.1

Reply:
Fixed